client/src/templates/Challenges/classic/classic.css

```diff
@@ -109,13 +109,12 @@
 }
 
 #mobile-layout-pane-instructions:focus-visible {
-  outline: 3px solid var(--blue-mid);
   outline-offset: -2px;
 }
 
 @supports not selector(:focus-visible) {
   #mobile-layout-pane-instructions:focus {
-    outline: 3px solid var(--blue-mid);
+    outline: 3px solid var(--focus-outline-color);
```

Comment:
I think we can delete `outline` completely since it is the same as the default.

Reply:
It's a catch for the focus-visible property not existing. We aren't going to have an outline if the `:focus-visible` isn't supported on that particular browser. Is this something everyone is okay with? 

That being said, I could very easily delete if we change the selector I made to a regular focus one.

Reply:
Yes, it's a fallback, but in this case the fallback is specific to `#mobile-layout-pane-instructions`. I have no problem having a global fallback just in case, but if we are going to have that then it should be in global.css right below where we create the default `:focus-visible` outline. Then the only thing we need in this fallback is the offset.

Reply:
Okay I believe I just resolved this